docker/compose/avalon-pool.yaml

```diff
@@ -74,6 +76,8 @@ services:
         - https_proxy
         - no_proxy
         - ENCLAVE_TYPE=wpe
+        - IMAGE=$IMAGE
```

Comment:
Would not be required. Will be taken up from default docker-compose.yaml.

Reply:
Yes. Removed it along with other duplicate attriutes.